data/sql/updates/pending_db_world/rev_1611782665942053526.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611782665942053526');
+
+-- Lower respawn rate of The Book of Ur
+￼
```

Comment:
dafuq O.o

Reply:
?

Reply:
You see the little weird character on L#4

Reply:
i don't see it but there seems to be one when i edit the file o_0

Reply:
Could you check again? I don't see it... but I tried to fix the weird empty line...